api/GlobalEventHandlers.json

```diff
@@ -3618,7 +3618,7 @@
               "version_added": true
             },
             "ie": {
-              "version_added": true
+              "version_added": false
```

Comment:
http://mdn-bcd-collector.appspot.com/tests/api/Window/onresize tests true in IE11, so there's a problem with the methodology.

Reply:
Hmm, it's available on the `Window` API, but not on any of the other APIs.  Should we just add a note, or `partial_implementation` with a version number?

Reply:
I think we should split mixins into their interfaces which would make partial impl. statements in this situation unnecessary. For now, how about just setting it to the version from when it was supported on any interface?

It sort of doesn't matter what we do since all mixin data will have to be revisited later anyway.

Reply:
Sounds good -- I added partial implementation statements mentioning how it's only available on `Window` in IE and EdgeHTML, so that we have the data in here for whatever we'll end up doing with mixins!